src/serve/javaEmulators.ts

```diff
@@ -73,6 +85,11 @@ function _getCommand(emulator: JavaEmulators, args: { [s: string]: any }): JavaE
 
   const cmdLineArgs = baseCmd.args.slice();
   Object.keys(args).forEach((key) => {
+    if (baseCmd.optionalArgs.indexOf(key) < 0) {
```

Comment:
Why this change? I'd suggest that we just error when we see an unsupported flag.

Reply:
This is the first time we want to add a flag to the in-memory `FooEmulatorArgs` that we don't want to pass directly to the JAR (`auto_download`).  So this filters those out.